### ISSUE TITLE
Add custom Headers/Footers to `publish`

### ISSUE THREAD

**Issue by robertberrington** [doorstop-dev/doorstop#166]
We have a project that is under Export Restrictions/ITARs ... We need to mark all documentation as restricted.

Currently the `doorstop publish` provides no functionality to mark the "printed version" with any info.

Use case could be:

```
doorstop publish --header="My Company, My Address" --footer="Copyright Year, Restrictions"
```
##

**Comment by michaelnt** [comment]
This is probably addressed by #225 which allows a custom template which could contain the footer.

If exporting to word you could possibly make use of a MD template to add additional information before or after the doorstop data, but probably not a footer. The footer could be added in either the word or latex template of using pandoc

**Comment by neerdoc** [comment]
Closing this issue. This can now be solved with templates for markdown publish or by LaTeX publish.
